gtk3/sample/misc/textbuffer_serialize.rb

```diff
@@ -20,100 +20,106 @@
 textview.set_size_request(600, 400)
 buffer = textview.buffer
 buffer.text = DATA.read
+deserialize_format = buffer.register_deserialize_tagset(nil)
 format = buffer.serialize_formats[0]
-buffer.register_serialize_tagset(nil)
-buffer.register_deserialize_tagset(nil)
 
 window = Gtk::Window.new("Gtk::TextBuffer Serialize Demo")
 vbox = Gtk::Box.new(:vertical, 0)
 
-serialize_button = Gtk::FileChooserButton.new("Serialize to a file", 
-                                              Gtk::FileChooserAction::OPEN)
-deserialize_button = Gtk::FileChooserButton.new("Serialize to a file", 
-                                                Gtk::FileChooserAction::OPEN)
-
-serialize_button.current_folder = current_folder
-deserialize_button.current_folder = current_folder
-
 toolbar = Gtk::Toolbar.new
-
-toolbar.append(Gtk::Stock::OPEN, "Deserialize from a file") do
-  dialog = Gtk::FileChooserDialog.new("Deserialize from a file",
-                                      window,
-                                      Gtk::FileChooserAction::OPEN,
-                                      nil,
+toolbar.set_style(Gtk::ToolbarStyle::BOTH)
+button_open = Gtk::ToolButton.new(:icon_widget => nil, :label => "Deserialize from a file",:stock_id => Gtk::Stock::OPEN)
+button_open.signal_connect "clicked" do
+  dialog = Gtk::FileChooserDialog.new(:title => "Deserialize from a file",
+                                      :parent => window,
+                                      :actions => Gtk::FileChooserAction::OPEN, #or :open
```

Comment:
Yes. To be more precise, nick name of GEnum or GFlags by Symbol is better.
(`Gtk::Stock::OPEN` isn't good example because it's just a macro not GEnum.)

For example, both `Gtk::Window.new(Gtk::WindowType::POPUP)` and `Gtk::Window.new(:popup)` work. We want to recommend the latter. Because users don't need to remember enum namespace (`Gtk::WindowType` in this example).